src/nl/surf/eduhub_rio_mapper/api.clj

```diff
@@ -41,6 +41,24 @@
            [java.net MalformedURLException URL]
            [org.eclipse.jetty.server HttpConnectionFactory]))
 
+(def server-stopping (atom false))
+
+(def nr-active-requests (atom 0))
+
+(defn wrap-request-counter [app]
```

Comment:
deze naam is niet heel duidelijk.  `wrap-active-request-counter` of zoiets is denk ik duidelijker.